src/test/java/StringCalculatorAssertTest.java

```diff
@@ -0,0 +1,46 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("String Calculator Assert Test")
+public class StringCalculatorAssertTest {
+
+    private StringCalculator calculator = new StringCalculator();
+
+    @Test
+    @DisplayName("빈 문자열을 입력하면 0을 반환한다.")
+    void input_empty_string() {
+        String input = "";
+
+        // 인자를 비교하는 다양한 방법
+        assertEquals(calculator.calculate(input), 0);
+        assertThat(calculator.calculate(input)).isZero();
+        assertThat(calculator.calculate(input)).isEqualTo(0);
```

Comment:
AssertJ와 JUnit5를 함께 사용하셨네요!
4단계 미션은 Junit5로 작성된 테스트를 AssertJ를 사용한 테스트로 바꾸는 요구사항이었어요!
JUnit5로 작성된 테스트 코드를 AssertJ로 바꿔주세요!
그리고 AssertJ로 바꾸면서 어떤 점을 느꼈는지 알려주세요!

Reply:
또한 검증에 사용할 결과는 하나만 사용해도 충분할 것 같아요!
```suggestion
        int actual = calculator.calculate(input);
        assertEquals(actual, 0);
        assertThat(actual).isZero();
        assertThat(actual).isEqualTo(0);
```